### PR TITLE
[itests mqtt.homeassistant] Fixed homeassitant itests with less impact to main build

### PR DESCRIPTION
**Pull request by mehoekstra**
This pull request allows the mqtt.homeassistant itests to function.

This replaces pull request #7140.  That test inappropriately changed the behavior of all itests such that in the itest.bundrun files --runbundles was ignored and OSGI dependencies were computed at build time on the CI machine (Jenkins or Travis).  This pull request changes that behavior only locally in the mqtt.homeassistant test.

Squashed commits and rebased at 6:14 AM March 27th GMT.   Travis tests pass for all openhab-addons bundles.  Full Jenkins build 18040 completed successfully as well https://ci.openhab.org/job/PR-openHAB2-Addons/18040/.  Artifacts here: https://openhab.jfrog.io/openhab/webapp/#/builds/PR-openHAB2-Addons/18040/1585334263515

### CODE REVIEW

**Comment by mehoekstra** [comment]
@J-N-K @wborn @kaikreuzer @cpmeister can you take another look?  I believe I have resolved the issue that caused the Jenkins build failure the other day.  As of now, 9:00PM GMT March 17th, this PR is synced with the latest 2.5.x branch in the main repository

File: itests/pom.xml [hunk 1]
```diff
@@ -117,6 +119,11 @@
           <type>pom</type>
           <scope>provided</scope>
         </dependency>
+        <dependency>
+		    <groupId>biz.aQute.bnd</groupId>
+		    <artifactId>biz.aQute.tester</artifactId>
+		    <version>${bnd.version}</version>
+		</dependency>
```
Comment (cpmeister) [inline itests/pom.xml]: got some weird indentation going on

Comment (cpmeister) [reply itests/pom.xml]: See if you can format this whole file.

Comment (cpmeister) [reply itests/pom.xml]: Actually, just format the parts that you are changing. Don't want to make this more difficult than necessary for other reviewers.

Comment (mehoekstra) [reply itests/pom.xml]: OK, I fixed the formatting and rebased again against the current openhab-addons/2.5.x branch as of 3:45AM March 19th GMT.

**Comment by mehoekstra** [comment]
@cpmeister, @J-N-K, @wborn, anything else you would like to see changed?

**Pull request closed without merging**